provider/testdata/Cargo.toml

```diff
@@ -71,6 +72,10 @@ locales = [
     # - Week of month/year have plural variants.
     "fil",
 
+    # Hebrew:
+    #  - Has context dependent list fragments 
+    "he",
```

Comment:
Nit: Add Hebrew list pattern data as a special case below, similar to cs/haw date formatting data.